README.md

```diff
@@ -1 +1,191 @@
+  deny_deleting_amazon_VPC_flowlogs = true
+
+  # Require tags on resources
+  require_tag_on_specific_resources = true
+  actions                           = ["ec2:RunInstances"]
+  resources                         = ["arn:aws:ec2:*:*:instance/*"]
+  resources_tag = [{
+      test     = "Null"
+      variable = "Owner"
+      values   = ["true"]
+  }]
+
+  # Require Instance Metadata Service Version 2
+  require_IMDSv2 = true
+
+  # For denying modification of S3 public access 
+  deny_modifying_S3_Block_Public_Access  = true
+  deny_s3_bucket_public_access_resources = ["arn:aws:s3:::testbucket"]
+
+  # Deny VPC modification and deletion
+  deny_vpc_modification = true
+
+  deny_modifying_IAM_password_policy = true
+
+  deny_creation_savings_plans        = true
+  deny_purchasing_reserved_instances = true
+
+  # Allow only specific instance types ec2 to create
+  allow_only_approved_Ec2_instance_types = true
+  allowed_ec2_instance_types             = ["*.nano"]
+
+  deny_creating_iam_access_keys = true
+
+  # Specify target on which policies will be imposed (like OU's or specific account ids)
+  targets                       = var.ou_targets
+}
+```
+
+SCP Policies 
+------
+| Name | Description |
+|------|--------|
+| deny_only_approved_services | Policy for denying only a set of services only |
+| deny_root_user_access | Access to root account will be not allowed |
+| region_enforcement |  Allow creating resources in a specific region. |                       
+| deny_ability_to_leave_Organization |  Deny child account to leave organization. |          
+| deny_ability_to_modify_specific_IAM_role | Deny modification of specific IAM Role. |    
+| deny_deleting_amazon_VPC_flowlogs | Deny deletion of VPC flowlogs. |        
+| deny_resource_creation_with_no_tag | Specify the resources which should have tags imposed |        
+| require_IMDSv2 | Require ec2 to use IMDSv2 |            
+| deny_creation_of_unencrypted_ebs_volume | Deny creation of unencrypted ebs. | 
+| deny_RDS_unencrypted | Deny creation of unencrypted RDS |        
+| deny_unencrypted_object_uploads_statement | Deny uploading unencrypted data in S3. | 
+| deny_modifying_S3_Block_Public_Access  | Deny modification of S3 public access. |      
+| deny_vpc_modification | Deny creation and modification in vpc. |               
+| deny_modifying_IAM_password_policy  | Deny changes to the IAM password policy. |     
+| deny_creation_savings_plans |  Deny creation of savings plans. | 
+| deny_purchasing_reserved_instances | Deny purchasing of reserved instances. | 
+| allow_only_approved_Ec2_instance_types  | Allow creation of specific ec2 instance type |  
+| deny_creating_iam_access_keys  | Deny creating iam access keys and iam user. | 
+
+# Inputs :
+
+General Variables
+------
+| Name | Description | Type | Default | Required |
+|------|-------------|------|---------|:--------:|
+| name | Name to be assigned on the policy | `string` | "" | Yes
```

Comment:
scp_policy_name
remove type var.  type should ba hardcoded SERVICE_CONTROL_POLICY
scp_targets
scp_tags

Reply:
Updated it 
And removed the variable for type

Reply:
Updated it 
And removed the variable for type